File: Algorithms/Mathematics/Sum_of_n_natural_numbers/Formula/cpp/formula.cpp
#include <iostream>

unsigned long sumNNumbers(unsigned int n) {
    return n*(n+1)/2;
}

int main() {
    std::cout << "Enter the number: ";
    unsigned int n;
    std::cin >> n;
    std::cout << "The sum of " << n << " natural numbers is: " << sumNNumbers(n) << std::endl;

    return 0;
}

File: Algorithms/Mathematics/Sum of natural numbers/Formula/java/Formula.java
import java.util.Scanner;

public class Formula {
    public static long sumNNumbers(int n) {
        return n*(n+1)/2;
    }

    public static void main(String[] args) {
        System.out.print("Enter the number: ");
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        sc.close();
        System.out.println("The sum of " + n + " natural numbers is: " + sumNNumbers(n));
    }
}
